sdks/java/core/src/test/java/org/apache/beam/sdk/io/AvroIOTest.java

```diff
@@ -512,6 +513,7 @@ public GenericClass apply(Long i) {
     }
 
     @Test
+    @Ignore("https://issues.apache.org/jira/browse/BEAM-6352")
```

Comment:
Should this bug be fixed in this PR? What's left?

Reply:
I missed it.